src/PrestaShopBundle/Resources/config/services/core/cldr.yml

```diff
@@ -0,0 +1,8 @@
+services:
+    _defaults:
+        public: true
+
+    prestashop.core.cldr.repository:
```

Comment:
repository is already registered under `prestashop.core.localization.cldr.repository` key, so you can use it.

or you can make alias since this one is using key that matches namesapce.